ledger-core/server/server.go

```diff
@@ -22,6 +22,23 @@ func NewNode(cfg configuration.Configuration) Server {
 	return insapp.New(cfg, appFactory)
 }
 
+func NewControlledMultiServer(controller cloud.Controller, configProvider *CloudConfigurationProvider) *insapp.Server { // nolint:interfacer
```

Comment:
there are no interfaces here, so nolint is redundant.

Reply:
```
golangci-lint run
12
server/server.go:25:60: `configProvider` can be `github.com/insolar/assured-ledger/ledger-core/instrumentation/insapp.ConfigurationProvider` (interfacer)
13
func NewControlledMultiServer(controller cloud.Controller, configProvider *CloudConfigurationProvider) Server {
```

Reply:
so why not to have insapp.ConfigurationProvider here?

Reply:
ok. I use interface and that is why removed check